pandas/src/datetime_helper.h

```diff
@@ -20,6 +23,7 @@ npy_int64 get_long_attr(PyObject *o, const char *attr) {
   return long_val;
 }
 
+PANDAS_INLINE
 npy_float64 total_seconds(PyObject *td) {
   // Python 2.6 compat
   npy_int64 microseconds = get_long_attr(td, "microseconds");
```

Comment:
actually i think this function can be removed entirely as we don't need 2.6 compat any longer